### ISSUE TITLE
[a11y]: Remove AILabel's default aria-label

### ISSUE THREAD

**Issue by wkeese** [carbon-design-system/carbon#19255]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.81

### React version

18

### Automated testing tool and ruleset

IBM Equal Access accessibility checker - latest

### Assistive technology

JAWS etc.

### Description

Remove the default `aria-label` specified for AILabel.

There's no reason for a text-only button to have an `aria-label`.  Why should the screen-reader experience be different than the sighted experience?   In either case, they should see (or hear) that there's a button that's labeled as "AI".

In particular, "Show information" is an invalid `aria-label` because it doesn't contain the text from the original button, thereby losing information for screen reader users:

<img width="1242" alt="Image" src="https://github.com/user-attachments/assets/83d39aa0-5c48-4c20-97fd-839a6b2de7b2" />

However, the problem is not just limited to the story.  src/components/AILabel/index.tsx also sets an unnecessary and incorrect default value for `aria-label`:

```js
['aria-label']: ariaLabel = 'Show information',
```

This was partly addressed by #18204 and #18407, but only for kind=inline:

```jsx
<ToggletipButton
  className={aiLabelButtonClasses}
  label={kind === 'inline' ? '' : ariaLabelText}>
  <span className={`${prefix}--ai-label__text`}>{aiText}</span>
  {kind === 'inline' && (aiTextLabel || textLabel) && (
    <span className={`${prefix}--ai-label__additional-text`}>
      {aiTextLabel || textLabel}
    </span>
  )}
</ToggletipButton>
```

I don't see any reason to limit it to `kind=inline`.

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-ailabel--default

### Steps to reproduce

Just look at Storybook.

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by Mikadv** [comment]
The weird thing is that this Icon is English ONLY. 

We focus so much on a11y and i18n but this fails it twice.

In German it would be `KI` in Polish it would be `SI` and so on.

**Comment by wkeese** [comment]
I don't think there's anything special about this case.  The default value of `aiText` is "AI", but the application can (and must) change it for I18N support.

It's like how `Modal`'s`closeButtonLabel` has a default value of "Close", but the application can (and must) change it for I18N support.
